kong.conf.default

```diff
@@ -500,6 +500,18 @@
                                  #
                                  # See http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_prefer_server_ciphers
 
+#ssl_dhparam =                   # Defines DH parameters for DHE ciphers from the
+                                 # predefined groups: `ffdhe2048`, `ffdhe3072`,
+                                 # `ffdhe4096`, `ffdhe6144`, `ffdhe8192`, or
+                                 # from the absolute path to a parameters file.
+                                 #
```

Comment:
This comment doesn't make it clear to me what values the `ssl_dhparam` conf option can actually take. Is it one of [`ffdhe2048`, `ffdhe3072`,`ffdhe4096`, `ffdhe6144`, `ffdhe8192`], or is it something else?

> or from the absolute path to a parameters file

This seems to indicate that this config option can be set to a filepath. If that's the case, we should clarify what syntax and possible options to be put in that file.

The nginx link below does not clarify any of this either.

Reply:
yes, the value can be ffdhe2048, ffdhe3072, ffdhe4096, ffdhe6144, or ffdhe8192 (or actually any predefined group lua-resty-openssl/openssl knows about, but others are not worth to document) OR it can point to a file.